lib/linguist/languages.yml

```diff
@@ -4580,6 +4580,19 @@ XC:
   codemirror_mode: clike
   codemirror_mime_type: text/x-csrc
   language_id: 398
+XCompose:
+  type: data
+  extensions:
+  - ".XCompose"
+  - ".xcompose"
+  filenames:
+  - ".XCompose"
+  - ".XCompose.In"
```

Comment:
I could find any file with this filename: https://github.com/search?utf8=%E2%9C%93&q=filename%3A.xcompose.in+NOT+dhvdhgdkfn&type=Code&ref=searchresults